src/erlcloud_aws.erl

```diff
@@ -956,7 +976,9 @@ profiles_assume( Credential, Role,
 config_credential({Id, Secret}, Config) ->
     Config#aws_config{ access_key_id = Id, secret_access_key = Secret };
 config_credential({Id, Secret, Token}, Config) ->
-    Config#aws_config{ access_key_id = Id, secret_access_key = Secret, security_token = Token }.
+    Config#aws_config{ access_key_id = Id, secret_access_key = Secret, security_token = Token };
+config_credential(#aws_config{} = Config, _) -> Config.
```

Comment:
Am I right that in case of recursion `aws_config` is passed as first argument instead of second as in every other case?